### PR TITLE
Exclude backslashes from readability

### PR DESCRIPTION
**Pull request by noon-dawg**
This pull request excludes backslashes that are placed after periods from the readability analysis.

In some environments, a space followed by a backslash is used after a period to indicate a line break.

Example: `This is an example. /`

These backslashes are negatively impacting the readability score.

### CODE REVIEW

File: src/readability.ts [hunk 1]
```diff
@@ -366,6 +366,8 @@ export function preprocessMarkdown(markdown: string) {
             // Remove any new lines that are added for manual word wrapping.
             // Here we just presume these will be preceeded by a normal alphabetical character
             .replace(/([a-zA-Z])\n/g, '$1 ')
+            // Remove backslashes that are placed after a period
+            .replace(/\. \\\n/g, '.\n')
```
Comment (Weetbix) [inline src/readability.ts:370]: Looks good and safe to me! 😎 

Lets just add a test case to the readability test suite and we are good to go?

Comment (Weetbix) [reply src/readability.ts:370]: Ah, sorry I should have been more specific. The test documents aren't run in any type of automated tests. I was talking about adding test cases to this file: https://github.com/Rebilly/lexi/blob/main/src/readability.test.ts

I would remove your changes to`test-document` as it's a bit of an edge case. Hit me up on slack if you have any questions :)

Comment (noon-dawg) [reply src/readability.ts:370]: Ah, that seems obvious now! 🙈 

I added a test case [here](https://github.com/Rebilly/lexi/pull/476/files#diff-69bab8dfc28e6c1123d0e6a546d171fca13139550b715a35fa72671eade5433b)

**Review by Weetbix — APPROVED** [review]
Awesome 😎
